requirements.txt

```diff
@@ -13,8 +13,8 @@ python-coveralls==2.9.3
 tox==3.20.1
 pytest==6.1.2
 pytest-cov==2.10.1
-pytest_cases==2.5.0
+pytest_cases==2.7.1
```

Comment:
@vladdoster I'll leave this PR be since I believe you may still be looking at the issue with the build.  We can close it or leave it open, whichever is most appropriate.